compiler/test/compilable/ctod.i

```diff
@@ -0,0 +1,18 @@
+/*
+PERMUTE_ARGS:
+REQUIRED_ARGS: -o- -H -HF$(RESULTS_DIR)/compilable/ctod.di
```

Comment:
you want `-Hf=$(RESULTS_DIR)/compilable/ctod.di`

Reply:
also `-H` is redundant with `-Hf=`

Reply:
done